config/manager/kustomization.yaml

```diff
@@ -12,5 +12,4 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: docker.io/mellanox/network-operator
-  newTag: latest
+  newName: mellanox/network-operator
```

Comment:
Should we refer to NGC repo here?

Reply:
Updated. It is changed when running `make bundle`